manifest.schema.yaml

```diff
@@ -13,6 +13,12 @@
         },
         "integration": {
           "type": "string"
+        },
```

Comment:
Before merging this, please also prepare a PR that fixes the manifests in https://github.com/autokitteh/samples

Reply:
https://github.com/autokitteh/samples/pull/37